internal/deviations/deviations.go

```diff
@@ -597,6 +597,12 @@ func ISISCounterPartChangesUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetIsisCounterPartChangesUnsupported()
 }
 
+// GRIBISkipFibFailedTrafficForwardingCheck returns true for devices that do not
```

Comment:
comment has to match the function name, too :)

Reply:
updated it now. thank you.